### PR TITLE
Module 5 components part2

### PR DESCRIPTION
**Pull request by SergBlack**

### CODE REVIEW

File: src/components/MovieCard/components/DescContent.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+const Paragraph = styled.div`
+  margin: 0;
+`;
+
+const DescContent = ({ title, genres, releaseDate }) => (
+  <DescContainer>
+    <div>
+      <Title size="20px" content={title || 'No title set'} />
+      <Paragraph>{genres || 'No genres set'}</Paragraph>
```
Comment (mmarkelov) [inline src/components/MovieCard/components/DescContent.jsx]: maybe it's better to put 'No genres set' into a constant?

File: src/forms/EditMovieForm.jsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
+import React, { useState, useEffect } from 'react';
```
Comment (mmarkelov) [inline src/forms/EditMovieForm.jsx:1]: This is pretty close to `AddMovieForm` component. Maybe you can use some common component for this?

File: src/components/MovieCard/components/DescContent.jsx (hunk 1 above)
Comment (SergBlack) [reply src/components/MovieCard/components/DescContent.jsx]: Thanks! Its a good idea.

File: src/forms/EditMovieForm.jsx (hunk 2 above)
Comment (SergBlack) [reply src/forms/EditMovieForm.jsx:1]: Thanks. I think about and refactor)

**Pull request merged**